samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_es.properties

```diff
@@ -306,6 +313,7 @@ how_content_4=-Debajo de <b>Comentarios >Publicar las preguntas y los siguientes
 how_content_5=-Haga clic en el bot\u00f3n <b>Guardar Configuraci\u00f3n</b>.
 
 applyGradesDesc=a todos los participantes "Sin entrega". 
+applyGradesDescAvg=a todos los participantes con 0 env\u00edos.
```

Comment:
¿a todos los participantes sin envíos.?

Reply:
This is a little weird... the text "applyGradesDescAvg" only appers when the Assessment is grading by average score.
See line: https://github.com/sakaiproject/sakai/blob/master/samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp#L201
At the moment, I have put the same translation for both: "a todos los participantes "Sin entrega". "